design-system/skyyrose-storefront/src/styles/commercial-polish.css

```
@@ -0,0 +1,383 @@
+:root {
+    /* Spacing scale — generous breathing room for premium feel */
+    --cp-space-xs: clamp(0.5rem, 1vw, 0.75rem);
+    --cp-space-sm: clamp(0.75rem, 1.5vw, 1rem);
+    --cp-space-md: clamp(1rem, 2vw, 1.5rem);
+    --cp-space-lg: clamp(1.5rem, 3vw, 2.25rem);
+    --cp-space-xl: clamp(2rem, 5vw, 3.5rem);
+    --cp-space-2xl: clamp(3rem, 7vw, 5.5rem);
+    --cp-space-3xl: clamp(4rem, 10vw, 8rem);
+
+    /* Elevation — soft, layered shadows that suggest depth without weight */
+    --cp-shadow-xs: 0 1px 2px rgba(10, 10, 10, 0.08);
```

Stylelint (17.13.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)